src/rpc/rpc.cc

```diff
@@ -69,10 +69,26 @@ RPCStatus SendRPCMessage(const RPCMessage& msg, const int32_t target_id) {
 }
 
 RPCStatus RecvRPCMessage(RPCMessage* msg, int32_t timeout) {
-  // ignore timeout now
-  CHECK_EQ(timeout, 0) << "rpc cannot support timeout now.";
-  RPCContext::getInstance()->receiver->Recv(msg);
-  return kRPCSuccess;
+  static constexpr int32_t default_timeout = 5 * 1000;  // milliseconds
+  RPCStatus status;
+  const int32_t real_timeout = timeout == 0 ? default_timeout : timeout;
+  do {
+    status = RPCContext::getInstance()->receiver->Recv(msg, real_timeout);
+    if (status == kRPCTimeOut) {
+      static const std::string log_str = [real_timeout, timeout]() {
+        std::ostringstream oss;
+        oss << "Timed out when trying to receive rpc message within "
+            << real_timeout << " milliseconds.";
+        if (timeout == 0) {
+          oss << " Retrying to receive as timeout is configured as 0 which "
+                 "means waiting infinitely.";
```

Comment:
`oss << "Receiving messages timeout. Retrying."`.

Reply:
I think changing to this will be a bit duplicate as we have outputted such info in the previous line.

Reply:
For L79-L84, how about:
```c++
std::ostringstream oss;
oss << "Recv RPCMessage timeout in " << real_timeout << " ms."
    << (timeout == 0? " Retrying ..." : "");
```

Reply:
ok